test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/SubWindowTest.cs

```diff
@@ -1,102 +1,142 @@
+
+        public void Activate()
+        {
+            Initialize();
+        }
+
+        public void Deactivate()
+        {
+        }
+
+        private void Initialize()
         {
             mainWin = NUIApplication.GetDefaultWindow();
             mainWin.KeyEvent += OnKeyEvent;
             mainWin.BackgroundColor = Color.Cyan;
-            mainWin.WindowSize = new Size2D(500, 500);
+            mainWin.WindowSize = new Size2D(5 * testSize, 5 * testSize);
             mainWin.TouchEvent += WinTouchEvent;
             belowParent = false;
 
-            TextLabel text = new TextLabel("Hello Tizen NUI World");
-            text.HorizontalAlignment = HorizontalAlignment.Center;
-            text.VerticalAlignment = VerticalAlignment.Center;
-            text.TextColor = Color.Blue;
-            text.PointSize = 12.0f;
-            text.HeightResizePolicy = ResizePolicyType.FillToParent;
-            text.WidthResizePolicy = ResizePolicyType.FillToParent;
+            TextLabel text = new TextLabel("Hello Tizen NUI World")
+            {
+                HorizontalAlignment = HorizontalAlignment.Center,
+                VerticalAlignment = VerticalAlignment.Center,
+                TextColor = Color.Blue,
+                PointSize = testPointSize,
+                HeightResizePolicy = ResizePolicyType.FillToParent,
+                WidthResizePolicy = ResizePolicyType.FillToParent
+            };
             mainWin.Add(text);
 
-            Animation animation = new Animation(2000);
-            animation.AnimateTo(text, "Orientation", new Rotation(new Radian(new Degree(180.0f)), PositionAxis.X), 0, 500);
-            animation.AnimateTo(text, "Orientation", new Rotation(new Radian(new Degree(0.0f)), PositionAxis.X), 500, 1000);
+            Animation animation = new Animation(2 * oneSecond);
+            animation.AnimateTo(text, "Orientation", new Rotation(new Radian(new Degree(180.0f)), PositionAxis.X), 0, oneSecond);
+            animation.AnimateTo(text, "Orientation", new Rotation(new Radian(new Degree(0.0f)), PositionAxis.X), oneSecond, 2 * oneSecond);
             animation.Looping = true;
             animation.Play();
 
-            log.Fatal(tag, "animation play!");
+            log.Debug(tag, "animation play!");
         }
 
-        void CreateSubWin3()
+        private void CreateSubWinThird()
         {
-            if(subWin3)
+            if (subWinThird)
             {
-                log.Fatal(tag, $"Sub Window3 is already created");
+                log.Debug(tag, $"subWinThird is already created");
                 return;
             }
-            subWin3 = new Window("subWin3", new Rectangle(0, 0, 300, 300), false);
-            subWin3.BackgroundColor = Color.Blue;
+            subWinThird = new Window("subWinThird", new Rectangle(0, 0, 3 * testSize, 3 * testSize), false)
+            {
+                BackgroundColor = Color.Blue
+            };
             View dummy = new View()
```

Comment:
what about subWinThirdChildView? or simply ChildDummyView, ChildPlaceholder?

Reply:
Thank for review. I fixed it.